config/config.toml.example

```diff
@@ -105,6 +105,11 @@ max-server-connections = 4096
 # Whether new collations are enabled, as indicated by its name, this configuration entry take effect ONLY when a TiDB cluster bootstraps for the first time.
 new_collations_enabled_on_first_bootstrap = false
 
+# ignore-ddl-temporary-keyword is used to make ddl TEMPORARY keyword compatible with previous version TiDB.
+# turn on this option, 'TEMPORARY' will ignored by TiDB as previous. Please make sure you fully understand the negative impact.
+# turn off this option, use 'CREATE/DROP TEMPORARY TABLE' or 'CREATE/DROP TEMPORARY SEQUENCE' will reported an error.
+ignore-ddl-temporary-keyword = false
```

Comment:
Consider an older version of TiDB, which do not have this config item in the config file, updated to this version. The default value of this item is `false`, then there is chance that the application may failed to run on the newer version TiDB.

BTW, it makes user confused when we using double negation. How about renaming it to `reserve-ddl-temporary-keyword`?

Reply:
OK, I will change the `defaultConf.IgnoreDDLTemporaryKeyword` to `true`.

but, I cannot agree to your  renaming advice. `reserve-ddl-temporary-keyword` is  ambiguous compare with `ignore-xxx` for new user, `TEMPORARY` is standard SQL syntax, why need to reserve? set to `false` might be interpreted as

1.  no  `TEMPORARY` syntax supported for tidb, or 
2. just ignore `TEMPORARY`